modules/util/css-builder/src/main/java/com/liferay/css/builder/CSSBuilder.java

```diff
@@ -170,11 +142,15 @@ public void close() throws Exception {
 		_sassCompiler.close();
 	}
 
-	public void execute(List<String> dirNames) throws Exception {
+	public void execute() throws Exception {
 		List<String> fileNames = new ArrayList<>();
 
-		for (String dirName : dirNames) {
-			_collectSassFiles(fileNames, dirName, _docrootDirName);
+		String docrootDirName = _cssBuilderArgs.getDocrootDir().toString();
```

Comment:
Avoid chaining. Also, sure we have to do this back and forth between files and strings? Maybe we can simplify 😊